workflow/scripts/prioritization/prediction.py

```
@@ -244,126 +251,174 @@ def extract_epilens(lengths):
-            #     else:
-            #         seq += "%0A" + line.strip()
-            # fh.close()
-            #
-            # call = f"curl --data \"method=netmhciipan_ba&sequence_text={seq}"
-            # call += f"&allele={allele}&length={epilen}\""
-            # call += " http://tools-cluster-interface.iedb.org/tools_api/mhcii/"
-
-        # make sure that there are entries in the file
-        if os.stat(fa_file).st_size != 0:
+
+        result = subprocess.run(call,
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)